server/controllers/comment.js

```diff
@@ -104,6 +106,7 @@ class Comment {
     }
 
     const { slug, id } = request.params;
+
     const article = await articleRepo.getArticleBySlug(slug);
     if (article === null) {
```

Comment:
We have a checkIfArticleExists middleware. That could be implemented here, so the route gets intercepted if the article doesn't exist

Reply:
Oh. Thank you.